modules/virt-configuring-default-cpu-model.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-default-cpu-model.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-default-cpu-model_{context}"]
+= Configuring the default CPU model
+
+Configure the `defaultCPUModel` by updating the `HyperConverged` custom resource (CR). You can change the `defaultCPUModel` while OpenShift Virtualization is running.
```

Comment:
I think this should use the attribute `:VirtProductName:` right?
```suggestion
Configure the `defaultCPUModel` by updating the `HyperConverged` custom resource (CR). You can change the `defaultCPUModel` while {VirtProductName} is running.
```

Reply:
Indeed!